src/main/resources/application.properties

```diff
@@ -1,10 +1,12 @@
 # jdbc.X
-jdbc.driverClassName=com.mysql.jdbc.Driver
-jdbc.url=jdbc:mysql://localhost:3306/caesar
-jdbc.user=db_admin
-jdbc.pass=rNT5Vn[j(>R2r6.UXcJwtv];`RWewYH`
- 
+#jdbc.driverClassName=com.mysql.jdbc.Driver
+spring.datasource.url=jdbc:mysql://localhost:3306/caesar
+spring.datasource.username=db_admin
+spring.datasource.password=rNT5Vn[j(>R2r6.UXcJwtv];`RWewYH`
 # hibernate.X
-hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
-hibernate.show_sql=true
-hibernate.hbm2ddl.auto=validate
\ No newline at end of file
+spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.hibernate.ddl-auto=create
+spring.jpa.properties.hibernate.hbm2ddl.import_files_sql_extractor=org.hibernate.tool.hbm2ddl.MultipleLinesSqlCommandExtractor
+spring.jpa.properties.hibernate.hbm2ddl.import_files=caesar.sql,insert.sql
+spring.jpa.properties.hibernate.show_sql=true
```

Comment:
If we want those debug options to be checked into master, I'd say to go with those four:
```
spring.jpa.properties.hibernate.show_sql=true
spring.jpa.properties.hibernate.use_sql_comments=true
spring.jpa.properties.hibernate.format_sql=true
spring.jpa.properties.hibernate.type=trace
```